### ISSUE TITLE
Documentation problem RS_Stratification removed form ICES code lists

### ISSUE THREAD

**Issue by rix133** [ices-tools-dev/RDBES#107]
In the data model documentation the file  "RDBES Data model.xlsx" several tables (ie LO, FT, SA etc) declare a ices vocabulary code **RS_Stratification** that does not exist. Will it be added or changed to **YesNoFields** ?

**Comment by edvinf** [comment]
Issue relates to: https://github.com/ices-tools-dev/RDBES/issues/34

I hope we can get to revising the document shortly. If you need to make a tentative assumption in the mean time, assume it will be change to YesNoFields.

**Comment by HenrikK-N** [comment]
Yes, we will update this, when it is done the issue will be closed.
